packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -47,15 +50,20 @@ const Wrapper = styled.div`
   ${({ service }) => getSansRegular(service)}
   background-color: ${C_CONSENT_BACKGROUND};
   padding: ${GEL_SPACING_DBL} ${GEL_MARGIN_BELOW_400PX};
+  border-top: solid 0.125rem transparent;
 
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
     padding: ${GEL_MARGIN_ABOVE_400PX};
```

Comment:
The naming of this constant gives me the impression it is meant to be used for margin, not padding, this comment leads me to think this to: https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/spacings.js#L18
Is there a reason you chose to use this rather than `GEL_SPACING` that it essentially wraps?

Reply:
I think I instinctively used the margin constants because the padding changes a lot with breakpoints, but good spot. Happy to do this too 👍